package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "glob": "^7.1.3",
     "graceful-fs": "^4.1.2",
-    "handlebars": "^4.1.2",
+    "handlebars": "^4.5.1",
```

Comment:
Handlebars ^4.5.1 could still be vulnerable to Arbitrary Code Execution and Prototype Pollution. Please upgrade this to ```^4.5.3```

Reply:
Fixed, updated the PR, thanks for the observation.